src/wallet/wallet2.cpp

```diff
@@ -3084,6 +3289,10 @@ crypto::hash8 wallet2::get_short_payment_id(const pending_tx &ptx) const
 // take a pending tx and actually send it to the daemon
 void wallet2::commit_tx(pending_tx& ptx)
 {
+  // allow sending to subaddresses only on testnet
+  if (ptx.construction_data.dest_subaddr && !testnet())
+    throw std::runtime_error("Sending to subaddresses is allowed only on testnet at the moment!");
```

Comment:
You ask for testers, and hope someone tests :) I had some luck with the blockchain sync patch, but did not have much luck for other things. Nobody's tested multisig apart from one person who did one tx I think.